backend/src/main/java/com/webappsbusters/packagemanagement/mapper/ParcelMapper.java

```diff
@@ -0,0 +1,45 @@
+package com.webappsbusters.packagemanagement.mapper;
+
+import com.webappsbusters.packagemanagement.domain.Parcel;
+import com.webappsbusters.packagemanagement.domain.ParcelDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParcelMapper {
```

Comment:
It's ok, but you can try https://github.com/orika-mapper/orika, it should map fields automagically